shotgun_api3/shotgun.py

```diff
@@ -945,23 +945,39 @@ def followers(self, entity):
         
         return self._call_rpc('followers', params)
 
-    def schema_entity_read(self):
+    def schema_entity_read(self, project_entity=None):
         """Gets all active entities defined in the schema.
 
         :returns: dict of Entity Type to dict containing the display name.
         """
 
-        return self._call_rpc("schema_entity_read", None)
+        params = {}
+
+        if project_entity:
+                params["project"] = project_entity
```

Comment:
Just a heads up that in the github diff, this looks like it's may be indented inconsistently.